saleor/graphql/app/dataloaders/app.py

```diff
@@ -46,3 +46,16 @@ def batch_load(self, keys):
         )
 
         return [apps.get(authed_apps.get(key)) for key in keys]
+
+
+class ActiveAppByIdLoader(DataLoader):
+    context_key = "active_app_by_id"
+
+    def batch_load(self, keys):
+        apps_map = (
+            App.objects.using(self.database_connection_name)
+            .filter(is_active=True, removed_at__isnull=True)
+            .prefetch_related("permissions__content_type")
```

Comment:
We should avoid it. In this case, I decided to leave it as it is because I want to reuse `calculate_webhooks_for_multiple_events` and don't see any other place to reuse this data. Some time ago we also talked about removing `content_type` because we don't use it and it is possible due to Django Auth drop.